src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,35 +15,59 @@
 package com.google.sps.servlets;
 
 import com.google.sps.data.SmallCityService;
+import com.google.sps.data.Listing;
+import com.google.sps.data.MapLocation;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.LinkedList;
 import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.logging.Logger;
 
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   
   SmallCityService smallCityService;
+  private Logger logger = Logger.getLogger(DataServlet.class.getName());
```

Comment:
logger is not really a descriptive variable name, what does it log?

Reply:
I fixed the initalization to be a final static but I think LOGGER is descriptive enough because it can log anything and that is the conventional naming. Let me know if you disagree

Reply:
I think the naming convention is weird for me since the variable name is the same as variable type. Like you would never do int int or double double, so maybe you can change it to printer

Reply:
I know this is not the official style guide, but this is how the google development example handles this: https://cloud.google.com/logging/docs/setup/java#example_2

Reply:
sounds good then, thank you for the clarification